rtublen/TODO
# Add ability to set task to completed

# Description 

Added way to set task to completed, might not work as intended but works as described.

## Testability

1. Copy and clone to local repository
2. Start application and verify that we can add a task, and then in menu option "List tasks",  set that task to completed by pressing "C" followed by the Id of the task.
